src/main_dpdk.cpp

```diff
@@ -5951,6 +5978,24 @@ COLD_FUNC int update_global_info_from_platform_file(){
         }
     }
 
+    if (cg->m_latency_measurement.length()) {
+        if ((strcasecmp(cg->m_latency_measurement.substr(0, 5).c_str(), "NANOS") == 0) ||
+            (strcmp(cg->m_latency_measurement.c_str(), "1") == 0)) {
+            g_opts->m_latency_measurement = CParserOption::LATENCY_METHOD_NANOS;
```

Comment:
If we have std::string i think there is no need to fallback to posix functions.

Reply:
The problem is `strcasecmp` does not really have its counterpart in C++ world.  The `str::transform` solution is much less readable, therefore I do not think it is a good fit here.  Moreover `str::transform` is never used in TRex code.

As for simple `strcmp` I think going for `.compare()` would not be a good option – mixing Posix functions with `str::` would introduce kind of mess.